### DISCUSSION TITLE
Dynamic Prompts / Wildcards

### DISCUSSION THREAD

**Discussion by DefineSleep** [comfyanonymous/ComfyUI#862] [Q&A] [answered]
is there a way to use dynamic prompts (wildcards) like in the WebUI, having a text file with prompts on them and every loop it picks one?
i know we can do {red|green|blue} but what if i have 100+ prompts per file. is there a way to call out the full .txt

**Comment by ltdrdata** [comment, marked as answer]
There is multiple wildcard nodes. 
And this is one of them.
https://www.youtube.com/watch?v=joNVYSIzA4k
